packages/mock-app/src/components/Header/Header.tsx

```diff
@@ -57,22 +66,29 @@ function Header() {
 
   useEffect(() => {
     const path = location.pathname;
-    const getNameLinkFromSesion = sessionStorage.getItem('nameLink');
-
-    if (getNameLinkFromSesion) {
-      setHeaderBrandInfo({
-        name: getNameLinkFromSesion,
-        assets: {
-          logo: '',
-        },
-      });
-      sessionStorage.removeItem('nameLink');
-    }
+    const nameLink = convertPathToString(path ?? '');
+    const subAppStyles =
+      appConfig.apps.find((app) => app.name.toLocaleLowerCase() === nameLink.toLocaleLowerCase()) ??
+      appConfig.generalFeatures.find((app) => app.name.toLocaleLowerCase() === nameLink.toLocaleLowerCase());
+
+    setHeaderBrandInfo({
+      name: convertPathToString(path ?? ''),
+      assets: {
+        logo: subAppStyles && 'assets' in subAppStyles ? subAppStyles?.assets?.logo : '',
+      },
+    });
+
```

Comment:
I think that when the page navigates to the 404 route, the title in the header should display the value of `appConfig.name` instead of 404.

<img width="1317" alt="Screenshot 2024-06-13 at 2 15 17 PM" src="https://github.com/uncefact/tests-untp/assets/43271346/472350ca-d505-4193-bac9-65e15f69c95a">

Reply:
Thank you. Actually, I think we should create an error page for these cases instead of redirecting to 404 with empty content.